web/packages/teleport/src/config.ts

```diff
@@ -319,7 +319,7 @@ const cfg = {
     awsRdsDbRequiredVpcsPath:
       '/v1/webapi/sites/:clusterId/integrations/aws-oidc/:name/requireddatabasesvpcs',
     awsDatabaseVpcsPath:
-      '/webapi/sites/:clusterId/integrations/aws-oidc/:name/databasevpcs',
```

Comment:
oh, and this change was needed to get the dev web ui working for me with a remote teleport cluster. That seems to require the v1 prefix for some reason 🤷‍♂️